integreat_cms/gvz_api/utils.py

```diff
@@ -182,21 +182,22 @@ def __init__(
         api = GvzApiWrapper()
         self.ags = region_ags
         if (
-            not region_name
+            region_name
             and not region_ags
             and (best_match := api.best_match(region_name, region_type))
         ):
             self.ags = best_match["ags"]
 
-        self.name: str = ""
+        self.name: str = "" if not region_name else region_name
```

Comment:
```suggestion
        self.name: str = region_name or ""
```

Reply:
Thanks @cclauss! I moved the suggestions to #2823.